docs/_posts/2022-04-01-issue-31.md

```diff
@@ -27,6 +27,30 @@ We welcome your feedback and suggestions. Let us know if we missed anything inte
 
 *  A
 
+### HLSL Working Group
```

Comment:
Maybe just HLSL/DirectX? I think our news should be based on components/project, instead of meetings.

Reply:
Or alternatively, add it under  LLVM/Clang, since that's where these live now